packages/s/Sendbird.json

```
@@ -0,0 +1,27 @@
+  
+{
+  "name": "sendbird",
+  "version": "",
+  "description": "SendBird provides the chat API and SDK for your app enabling real-time communication among your users.",
+  "filename": "SendBird.min.js",
+  "keywords": [
+    "sendbird",
+    "chat"
+  ],
+  "author": "Sendbird, Inc.",
+  "homepage": "https://sendbird.com/",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sendbird/SendBird-SDK-JavaScript"
+  },
+  "npmName": "sendbird",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "SendBird.min*"
+      ]
+    }
+  ]
```

```suggestion
  "autoupdate": {
    "source": "npm",
    "target": "sendbird",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "SendBird.min*"
        ]
      }
    }
  ]
```
Please use the new `autoupdate` format documented in CONTRIBUTING.md